### PR TITLE
[LF] Freeze 1.16

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**

### CODE REVIEW

File: sdk/compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs [hunk 1]
```diff
@@ -71,6 +71,10 @@ version1_14 = Version V1 (PointStable 14)
 version1_15 :: Version
 version1_15 = Version V1 (PointStable 15)
 
+-- | Daml-LF version 1.15
```
Comment (paulbrauner-da) [inline sdk/compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:74]:
```suggestion
-- | Daml-LF version 1.16
```

File: sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala [hunk 2]
```diff
@@ -71,12 +71,12 @@ object LanguageVersion {
     val bigNumeric = v1_13
     val exceptions = v1_14
     val basicInterfaces = v1_15
+    val packageUpgrades = v1_16
+    val sharedKeys = v1_15
```
Comment (paulbrauner-da) [inline sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala:75]: I think you meant v1_16

File: sdk/language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/InterfacesTest.scala [hunk 3]
```diff
@@ -19,6 +19,9 @@ trait InterfacesTest
 
   import TestUtil._
 
+  protected override val cantonFixtureDebugMode: CantonFixtureDebugMode =
+    CantonFixtureDebugKeepTmpFiles
```
Comment (paulbrauner-da) [inline sdk/language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/InterfacesTest.scala:23]: Leftover?

**Review by paulbrauner-da — APPROVED** [review]
Thanks!

File: sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala (hunk 2 above)
Comment (remyhaemmerle-da) [inline sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala:75]:
```suggestion
    val sharedKeys = v1_16
```

File: sdk/language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/InterfacesTest.scala (hunk 3 above)
Comment (remyhaemmerle-da) [reply sdk/language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/InterfacesTest.scala:23]:
```suggestion
```

**Pull request merged**